### DISCUSSION TITLE
想请问下怎么切回1.5.7的ui

### DISCUSSION THREAD

**Discussion by Cyrker** [zdz/ServerStatus-Rust#146] [Q&A] [answered]
更新了1.6.2之后发现没有更换主题的位置了..
想尝试用nginx反代来更换，但是没找到1.5.7版本的主题文件

**Comment by zdz** [comment, marked as answer]
https://github.com/zdz/ServerStatus-Rust/releases/tag/v1.5.7 下载 assets 里面的 source code
